docs/getting_started/client.md

```diff
@@ -151,13 +158,52 @@ For a given session, this cache can be written and reloaded into an OPTIMADE cli
 !!! info
     In a future release, this cache will be automatically restored from disk and will obey defined cache lifetimes.
 
-### Counting entries and limiting results
+### Querying other endpoints
+
+The client can also query other endpoints, rather than just the default `/structures` endpoint (including provider-specific `extensions/<example>` endpoints).
+
+In the CLI, this is done with the `--endpoint` flag.
+In the Python interface, the different endpoints can be queried as attributes of the client class or equivalently as a paramter to `client.get()` or `client.count()` (see below).
+
+=== "Command line"
+    ```shell
+    optimade-get --endpoint "structures"
+    optimade-get --endpoint "references"
+    optimade-get --endpoint "info"
+    optimade-get --endpoint "info/structures"
+    optimade-get --endpoint "extensions/properties"
```

Comment:
I am not sure it is worthwhile to mention this, as there is just one database that supports "extensions/properties".

Reply:
I mention it in the paragraph above, but will expand on it if it is not clear

Reply:
Yes you are right, I should have read it better.